com.unity.render-pipelines.universal/ShaderLibrary/ShaderQuality.hlsl

```diff
@@ -0,0 +1,28 @@
+#ifndef UNIVERSAL_SHADER_OPTIONS_INCLUDED
+#define UNIVERSAL_SHADER_OPTIONS_INCLUDED
+
+//#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/ShaderOptions.hlsl"
+//use in some shader to compile properly:
+//#pragma multi_compile SHADER_OPTIONS_LOW SHADER_OPTIONS_MEDIUM SHADER_OPTIONS_HIGH
+
+#if !defined (SHADER_QUALITY_LOW) && !defined (SHADER_QUALITY_MEDIUM) && !defined (SHADER_QUALITY_HIGH)
+#if defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH)
```

Comment:
minor: this could be rewritten as if / elif / else instead of nested ifs to improve readability.

Reply:
The first if statement don't have any else statement.

Was this what you meant?:
#if !defined (SHADER_QUALITY_LOW) && !defined (SHADER_QUALITY_MEDIUM) && !defined (SHADER_QUALITY_HIGH) && defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH) && defined(SHADER_API_GLES)
#define SHADER_QUALITY_LOW
#elif !defined (SHADER_QUALITY_LOW) && !defined (SHADER_QUALITY_MEDIUM) && !defined (SHADER_QUALITY_HIGH) && defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH)
#define SHADER_QUALITY_MEDIUM
#elif !defined (SHADER_QUALITY_LOW) && !defined (SHADER_QUALITY_MEDIUM) && !defined (SHADER_QUALITY_HIGH)
#define SHADER_QUALITY_HIGH
#endif

Reply:
No, I'm proposing a slight change to make it more readable as something like this instead of nested ifs
```suggestion
#if defined(SHADER_API_GLES)
#define SHADER_QUALITY_LOW
#elif defined(SHADER_API_MOBILE) || defined(SHADER_API_SWITCH)
#define SHADER_QUALITY_MEDIUM
#else
#define SHADER_QUALITY_HIGH
#endif
```

Reply:
This should have been fixed now